admin-manual/provisioning_phone2.rst

```diff
@@ -0,0 +1,294 @@
+del telefono alla rete. È infatti possibile inserire i MAC
+address di telefoni ancora imballati.
+
+In ogni caso l'inserimento degli indirizzi MAC dei telefoni può avvenire con
+le seguenti modalità:
+
+* Digitare o copiare l'indirizzo MAC da un foglio di
+  calcolo, fattura o altro documento
+
+* Scansionare le reti locali alla ricerca di apparati telefonici
+  supportati, già accesi e collegati in rete
+
+* Scansionare il codice a barre dell'indirizzo MAC del telefono tramite app
+  per smartphone "Scan & Play"
+ 
+
+Associazione dei telefoni agli utenti
+-------------------------------------
+
+La configurazione di un telefono è completa quando questo viene associato ad un
+utente.
+
+Ad ogni utente possono essere associati al massimo 8 dispositivi telefonici.
+
+|product| assegna ad ogni dispositivo associato all'utente una numerazione
+progressiva con il criterio seguente:
+
+* ``Interno Principale`` - telefono principale, per esempio ``201``
+
+* ``91+Interno Principale`` - telefono 2, per esempio ``91201``
+
+* ``92+Interno Principale`` - telefono 3, per esempio ``92201``
+
+* ...
+
+Tuttavia dal punto di vista degli utenti, l'Interno Principale è l'unico numero
+importante da ricordare.
+
+
+
+Azioni da compiere sui telefoni
+===============================
+
+.. note::
+
+    Consideriamo al **primo avvio** i telefoni nuovi, appena tolti dalla
+    scatola, o che hanno subito il ripristino dei valori di fabbrica e non sono
+    mai stati avviati.
+
+
+I telefoni al **primo avvio** sono già in grado di raggiungere |product| per
+ottenere la propria configurazione secondo i metodi supportati.
+
+L'unica azione da compiere in questi casi è collegare il cavo Ethernet con PoE
+(Power over Ethernet) al telefono. Qualora PoE non fosse disponibile sarà
+necessario collegare anche il cavo di alimentazione del telefono.
+
+.. warning::
+
+    Verificare la compatibilità dei telefoni con i metodi di provisioning
+    supportati. Leggere attentamente le sezioni seguenti
+
+Nel caso un telefono sia già usato è possibile predisporlo all'associazione con
+|product| tramite le procedure di **aggiornamento del firmware** e di
+**ripristino dei valori di fabbrica**. Entrambe le procedure sono disponibili
+tramite l'interfaccia web di amministrazione del telefono.
+
+
+Metodi di provisioning
+======================
+
+I telefoni possono accedere alla propria configurazione mediante i protocolli
+standard del web, HTTP o HTTPS (porta TCP 80 o 443).
+
+Quando il MAC address del telefono è inserito in |product| viene generato un
+URL (indirizzo) di provisioning. Per esempio: ::
+
+    https://mio.centralino.cloud/provisioning/1234567890.1234/{mac}.cfg
+
+Questo URL contiene un segreto (``1234567890.1234`` nell'esempio) che autentica
+ed identifica il dispositivo che ne farà uso.
+
+Per ottenere l'URL di provisioning il telefono al primo avvio
+può utilizzare due metodi, **RPS** e **DHCP**. 
+
```

Comment:
I'd move this paragraph to a "RPS details" pragraph, and replace the existing with "Here magic happens" :D

Reply:
Thank you for the review!

I think what you call "the RPS details" paragraph is actually this section. I wouldn't split it: the purpose of the whole section is giving an idea of how it works. It should provide details.